src/main/java/com/redhat/cloud/policies/app/model/Policy.java

```diff
@@ -120,7 +120,11 @@ public String getMtime() {
 
   @JsonbTransient
   public void setLastEvaluation(long lastEvaluation) {
-    this.lastEvaluation = new Timestamp(lastEvaluation);
+    if (lastEvaluation == 0) {
+      this.lastEvaluation = null;
+    } else {
+      this.lastEvaluation = new Timestamp(lastEvaluation);
```

Comment:
A small note - this class is incorrect. ``java.sql.Timestamp`` is most certainly not what one should use. Perhaps you wanted ``LocalDateTime`` instead? (not broken in this commit - this has been broken all along)

Reply:
Although I'm not sure why not just send the epoch ..

Reply:
I think we probably should use the epoch. Is easier to work with it and we are losing the timezone when convertir it to `Timestamp`.